### ISSUE TITLE
basemodulepath is written in [master] section. Mistake?

### ISSUE THREAD

**Issue by confiq** [theforeman/puppet-puppet#206]
When I'm using the following:
```puppet 
 class {
    '::puppet':
      server_common_modules_path => ['/etc/puppet/modules/X','/etc/puppet/modules/Y'],
```
module puppet writes config under [master] section
```
    basemodulepath   = /etc/puppet/modules/X:/etc/puppet/modules/Y
```
However, according to the [puppet docs](https://docs.puppetlabs.com/puppet/3.7/reference/dirs_modulepath.html#configuring-the-modulepath) it should be in _[main]_ section.
Even when I'm running command ```puppet config print modulepath``` configuration in [master] is not working and ```puppet module list``` ignores basemodulepath in  [master] 
This should be easy fix which even I might be able to fix it but the question is, is it a bug?

**Comment by domcleal** [comment]
:+1: sounds fine to move to `[main]`.

**Comment by domcleal** [comment]
Resolved in #213 and 447c744, thanks @confiq!
